servicebroker/lib/config/setting.js

```diff
@@ -1,5 +1,6 @@
-//'use strict';
-
+'use strict';
```

Comment:
The more I look at this, the more confused I get.

I would have just had the validation here as a separate function added to the object returned.
Let the user call validate and decide how to handle the success or failure.

I don't see why parse() is even there since it is called as part of the return.
You might as well just inline it and be done with it.